pages/common/mosquitto_sub.md

```diff
@@ -0,0 +1,15 @@
+# mosquitto_sub
+
+> A simple MQTT version 3.1.1 client that will subscribe to topics and print the messages that it receives.
+
+- Subscribe to all broker status messages publishing on `iot.eclipse.org` port 1885 and print published messages verbosely, by default hostname resolves to `localhost` and port `1883`:
+
+`mosquitto_sub -v -h "iot.eclipse.org" -p 1885 -t {{\$SYS/#}}`
+
+- Subscribe to the topic `sensors/temperature` information on `localhost` with Quality of Service (`QoS`) set to 1:
+
+`mosquitto_sub -t {{sensors/temperature}} -q {{1}}`
+
+- Subscribe to Hard disk (HD_NAME) temperature updates on multiple machines defined by `HOSTNAME`. This expects each machine to be publishing it's temperature to "sensors/machines/`HOSTNAME`/temperature/HD_NAME":
```

Comment:
I don't think this needs to be Hard disk temperatures, or the middle metric has to be hostname. The + is just a catchall for whatever metric comes in between. (Please correct me if I am wrong)

We can shorten this to - `Subscribe to multiple topics matching a given pattern. (+ takes any metric name):`